src/sql/src/normalize.rs

```diff
@@ -459,31 +459,45 @@ pub fn create_statement(
 macro_rules! generate_extracted_config {
     // No default specified, have remaining options.
     ($option_ty:ty, [$($processed:tt)*], ($option_name:path, $t:ty), $($tail:tt),*) => {
-        generate_extracted_config!($option_ty, [$($processed)* ($option_name, Option::<$t>, None)], $(
+        generate_extracted_config!($option_ty, [$($processed)* ($option_name, Option::<$t>, None, false)], $(
```

Comment:
I found the changes to this macro hard to review, in part because not all the parameters are documented in the doc comment at the top of the macro. Can you explain what `tail` and `allow_multiple` mean there?

Reply:
I added documentation for `AllowMultiple`! `$tail` is an internal implementation detail of the macro, though, so I left that one undocumented. 

Agreed this macro is _really_ hard to understand, but also we have at least two things that want to use `AllowMultiple`, so it seems worth just eating the complexity for now. I think ultimately we want to pull this into a proc macro that's easier to understand and more flexible, but don't want to bite that off right now.